FND/DatesAndTimes/FinancialDates.rdf

```diff
@@ -565,6 +565,20 @@ The recurrence start date can also be relative to the starting Date of the overa
 		<fibo-fnd-utl-av:explanatoryNote>Time intervals may be indefinite, meaning that their beginning is primordiality or their end is perpetuity, or both (eternity). This vocabulary assumes that indefinite time intervals exist and have some duration, but their duration is unknown.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-dt-fd;TimeOfDay">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-fd;TimeInstant"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasTimeValue"/>
+				<owl:onDataRange rdf:resource="&xsd;string"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>time of day</rdfs:label>
+		<skos:definition>explicit time, according to a clock</skos:definition>
+		<fibo-fnd-utl-av:usageNote>The representation should be the same as for xsd:dateTime, excluding the date component of the string value. The value of the hasTimeValue property corresponds to xsd:time in XML schema datatypes, which is prohibited from use in OWL due to ambiguity in its definition.</fibo-fnd-utl-av:usageNote>
```

Comment:
I think it should be something like:
    <fibo-fnd-dt-fd:hasTimeValue rdf:datatype="&xsd;string">11:00:00</fibo-fnd-dt-fd:hasTimeValue>
    <fibo-fnd-dt-bd:hasBusinessCenter rdf:resource="&fibo-fbc-fct-bci;New_York"/>

Reply:
Fixed and added a note about the business center in financial dates, since to include it would cause a circular reference; added the restriction for it in Debt on RateResetTimeOfDay, since it is clear that there should be a business center associated with it in that case.